src/CMakeLists.txt

```
@@ -84,6 +84,16 @@ add_subdirectory(corecel)
 add_subdirectory(orange)
 add_subdirectory(celeritas)
 
+if(CELERITAS_USE_VECGEOM)
```

```suggestion
if(CELERITAS_USE_VecGeom AND CELERITAS_USE_Geant4)
```

but since the spelling is incorrect, that means this code isn't actually being used so can be deleted.